lib/use_packwerk/private.rb

```diff
@@ -113,21 +115,26 @@ def self.move_to_pack!(pack_name:, paths_relative_to_root:, per_file_processors:
               origin_pathname
             end
           end
-          file_move_operations = file_paths.map do |origin_pathname|
-            FileMoveOperation.new(
+          file_move_operations = file_paths.flat_map do |origin_pathname|
+            file_move_operation = FileMoveOperation.new(
               origin_pathname: origin_pathname,
               destination_pathname: FileMoveOperation.destination_pathname_for_package_move(origin_pathname, package_location),
               destination_pack: package
             )
+            [
+              file_move_operation,
+              file_move_operation.spec_file_move_operation
+            ]
           end
           file_move_operations.each do |file_move_operation|
             Private.package_filepath(file_move_operation, per_file_processors)
-            Private.package_filepath_spec(file_move_operation, per_file_processors)
```

Comment:
This method was just constructing another `FileMoveOperation` and then calling `Private.package_filepath`, so this simplifies things a bit IMO.